torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -402,6 +402,14 @@ RegisterOperators reg(
            push(stack, a.cpu());
            return 0;
          }),
+     Operator(
```

Comment:
we should mark this op as having side effects

Reply:
Is there a way of doing that in the schema ? I added it to hasSideEffects() in ir.cpp

Reply:
oh I missed that lol. carry on.